components/rewards/RewardsFooter.js

```diff
@@ -0,0 +1,56 @@
+import { FontAwesome5 } from '@expo/vector-icons';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { View } from 'react-native';
+import Colors from '../../constants/Colors';
+import { Subhead } from '../BaseComponents';
+/**
+ * @prop
+ * */
+
+function RewardsFooter({ customer, isGuest }) {
+  // const points = parseInt(customer.points);
+  let string1 = '';
```

Comment:
Wait this is still the same. Please look into https://reactjs.org/docs/conditional-rendering.html#inline-if-else-with-conditional-operator from @thumn 's comment.

As an example in our code, look at `StoreDetailText` (around line 88) in `StoreCard.js`  where it prints the text depending on if rewards are accepted at a store.

Reply:
Then you don't need to have these strings at all and you can just have it directly in the text components.

Reply:
{snapOrEbtAccepted && (
            <EBTStatusBar>
              <FontAwesome5 name="check" size={10} color={Colors.darkerGreen} />
              <Caption color={Colors.darkerGreen}> EBT</Caption>
            </EBTStatusBar>
          )}